Rakefile

```diff
@@ -10,7 +10,6 @@ desc 'Builds the site locally'
 task :build do
   puts 'Building site.'
   sh 'PRODUCTION="YES" jekyll build --destination _gh-pages'
-  sh 'PRODUCTION="YES" jekyll build --destination _gh-pages'
```

Comment:
This command was repeated [for Netlify previews](https://github.com/artsy/artsy.github.io/pull/460) which I don't think are still functioning?